_sass/_index.scss

```diff
@@ -4,10 +4,11 @@
 @forward 'mixins/all';
 @forward 'components/all';
 
-@forward 'pages/home';
-@forward 'pages/what_is_login';
-@forward 'pages/who_uses_login';
 @forward 'pages/create_an_account';
+@forward 'pages/find_a_participating_post_office';
+@forward 'pages/home';
 @forward 'pages/not_found';
-@forward 'pages/policy';
 @forward 'pages/partners';
+@forward 'pages/policy';
+@forward 'pages/what_is_login';
+@forward 'pages/who_uses_login';
```

Comment:
👏 You added helpful docs _and_ alphabetized this section while also fixing the styling? MVP!